lib/MusicBrainz/Server/Data/URL.pm

```diff
@@ -99,7 +99,7 @@ my %URL_SPECIALIZATIONS = (
     'Loudr'               => qr{^https?://(?:www\.)?loudr\.fm/}i,
     'LyricEvesta'         => qr{^https?://lyric\.evesta\.jp/}i,
     'LyricsNMusic'        => qr{^https?://lyricsnmusic\.com/}i,
-    'LyricWiki'           => qr{^https?://lyrics\.wikia\.com/}i,
+    'LyricWiki'           => qr{^https?://lyrics\.fandom\.com/}i,
```

Comment:
Should this regex also match wikia links since we haven't changed them yet?

Reply:
Yes! It would be nice also to replace `wikia` with `fandom` in URL display by overriding `href_url` in [LyricWiki.pm](https://github.com/metabrainz/musicbrainz-server/blob/v-2019-01-22/lib/MusicBrainz/Server/Entity/URL/LyricWiki.pm), see [VIAF.pm](https://github.com/metabrainz/musicbrainz-server/blob/v-2019-01-22/lib/MusicBrainz/Server/Entity/URL/VIAF.pm) for example.

Reply:
Made the updates, tested locally and they seem to work, but let me know :)